multicluster/cmd/link.go

```diff
@@ -31,6 +31,8 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const clusterNameLabel = "multicluster.linkerd.io/cluster-name"
```

Comment:
Should we add this to the labels pkg [(`linkerd2/pkg/k8s`)](https://github.com/linkerd/linkerd2/blob/main/pkg/k8s/labels.go#L392) since it may have to be referenced in multiple parts of the code?

Reply:
we can easily move this to somewhere appropriate when it gets referenced from multiple places